src/auth/authApp.js

```diff
@@ -160,7 +160,10 @@ export function loadUserData() {
  * @param  {String} [redirectURL='/'] Location to redirect user to after sign out.
  * @return {void}
  */
-export function signUserOut(redirectURL: string = '/') {
+export function signUserOut(redirectURL: string = null) {
   window.localStorage.removeItem(BLOCKSTACK_STORAGE_LABEL)
-  window.location = redirectURL
+
+  if (redirectURL != null){
```

Comment:
The lint that is causing the tests to fail is looks like it is around here. Need a space after the closing parenthesis and we should use `!==` instead of `!=`.